filesystem/rootfs/docker/etc/daemon.json

```diff
@@ -0,0 +1,20 @@
+{
+  "max-concurrent-downloads": 20,
+  "log-driver": "json-file",
+  "log-level": "warn",
+  "log-opts": {
+    "max-size": "10m",
```

Comment:
max-size: 10m  seems to small for prodcution use. need to be set 100m or bigger.